docs/guides/deploy_service.md

```diff
@@ -1,24 +1,34 @@
 Deploying a service with the {{open_autonomy}} framework requires that you have an already available service configuration, which you could be a service created from scratch, fetched from the [IPFS](https://ipfs.io/), or already registered in the on-chain protocol.
 
-## What you will learn
+<figure markdown>
+![](../images/development_process_deploy_service.svg)
+<figcaption>Part of the development process covered in this guide</figcaption>
+</figure>
 
-In this guide, you will learn how to:
+## What you will learn
 
-* Create a local deployment for testing purposes of
-  * a service stored in your machine.
-  * a service available in the on-chain registry.
-* Create a cloud deployment of a service.
+This guide covers step 6 of the [development process](./overview_of_the_development_process.md). You will learn how to create and run deployments for local services (testing) and minted services.
 
-Before starting this guide, ensure that your machine satisfies the framework requirements and that you have followed the [set up guide](./set_up.md). As a result you should have a Pipenv workspace folder.
+You must ensure that your machine satisfies the framework requirements and that you have [set up the framework](./set_up.md#set-up-the-framework) and [a local registry](./set_up.md#set-up-the-local-registry). As a result you should have a Pipenv workspace folder with a local registry (`./packages`) in it.
 
 ## Local deployment
 
-Local deployments of a service are recommended to test your service before you publish it to a remote registry. Open a terminal and follow the steps below.
+This section covers the deployment of services being developed in the local registry. You can use such deployments to test your service before you mint it in the Autonolas Protocol.
 
-1. **Build the agents' image.** Navigate to the local folder of the service that you want to deploy, that is, the folder containing the `service.yaml` file, and build the Docker image of the service agents:
+1. **Fetch the service.** In the workspace folder, fetch the service from the local registry:
+    <!-- TODO: autonomy packages lock + push all should not be necessary here, but otherwise it cannot build the image. -->
+    ```bash
+    autonomy packages lock
+    autonomy push-all
+    autonomy fetch your_name/your_service:0.1.0 --local --service
+    ```
+
+    This step is required to have a separate runtime folder (`./your_service`), outside of the local registry.
+
+2. **Build the service agents' image.** Navigate to the service folder and build the Docker image of the service agents:
 
     ```bash
-    cd <service_folder>
+    cd your_service
```

Comment:
This section of the guide is easy to follow I believe. However, I could not complete the steps successfully since I didn't have a service built from the previous section (and I think I've followed all the steps!).